### ISSUE TITLE
Alt text field inheriting theme color from the Customizer

### ISSUE THREAD

**Issue by jp-imagines** [Automattic/wp-calypso#60410]
### Quick summary

The "Alt text (alternative text)" setting on image-type blocks (Image, Gallery, Cover, etc.) should display in the default black text. However, on some themes, it seems to be "inheriting" one of the colors set at My Site > Appearance > Customize > Colors & Backgrounds.

### Steps to reproduce

1. Activate Apostrophe 2 or Balasana. Unsure which other themes may also be affected.
2. Navigate to My Site > Appearance > Customize > Colors & Backgrounds and set the "Links and Buttons" (Apostrophe 2) or "Text Color" (Balasana) to a light color (like `#e3ece3`). Save changes.
3. Add an image to a post or page. This can be an Image block or any other block which has the "Alt text" setting.
4. In the right sidebar, add some alt text.

### What you expected to happen

The alt text should be in black font so that it is legible against the white background.

### What actually happened

On Apostrophe 2, the alt text is black unless you click in the textarea to edit.

On Balasana, the alt text is always the theme color.

<img width="260" alt="Markup 2022-01-24 at 13 55 51" src="https://user-images.githubusercontent.com/5630717/150848226-e03fb408-66ad-49fe-8099-62e77dd37890.png">

### Context

4715595-zen

### Simple, Atomic or both?

Atomic

### Theme-specific issue?

Apostrophe 2 and Balasana definitely affected. Unsure how many others are also affected.

### Reproducibility

Consistent

### Severity

Some (< 50%)

### Available workarounds?

No but the platform is still usable

### Workaround details

The only workaround would be to change the corresponding theme color in the Customizer, which may not be an option for some users. Some themes (like Balasana) are more affected, since the alt text is _always_ in the theme color, and some users are more affected if they have a light theme color selected.

**Comment by Greatdane** [comment]
Not just the alt text field, all fields that you can enter text within the editor. For example Excerpt;

<img width="357" alt="Screenshot 2022-02-24 at 15 25 35" src="https://user-images.githubusercontent.com/5075840/155469840-3ed8bf4c-53b6-40a6-9eff-67f4a7a15dae.png">

(If I do not highlight the text here, it is impossible to see).
